.env.dist

```diff
@@ -1,3 +1,4 @@
+API_URI=http://libero-api
```

Comment:
but this is not read in https://github.com/libero/browser/blob/master/docker-compose.yaml#L15?

Reply:
Was thinking this is a form of documentation about what can be set generally (ie not just for the local Docker Compose setup), but maybe that's best kept elsewhere.